.changelog/2463.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_alert_configuration: Fixes nil pointer dereference when alert is modified outside Terraform
```

Comment:
Please put a period at the end of the sentence.

Reply:
Also, pleas consider rewriting as:

Fixes an issue where the `terraform apply` command crashes if you attempt  to edit an existing Terraform alert by adding a value to  `threshold_config`.